APILinkerGraphQL/Sources/Downloader/Options.swift

```diff
@@ -0,0 +1,51 @@
+import ApolloCodegenLib
+import Foundation
+
+public var endpoint: URL {
```

Comment:
Why is it public? Are you gonna use it outside?! 
Moreover, global constants doesn't look clean. I'd prefer enum with static properties, like
```swift
enum Constants {
  static var endpoint: URL = //
}
```